exporter/loadbalancingexporter/README.md

```diff
@@ -126,7 +127,9 @@ processors:
 
 exporters:
   loadbalancing:
-    routing_key: "service"
+    routing_key: "resource"
```

Comment:
Did you try this example? I believe this doesn't work, as the component is used in a traces pipeline as well, which doesn't seem to support routing per resource keys?